### ISSUE TITLE
Type errors in generated TS file

### ISSUE THREAD

**Issue by ryb73** [timkendall/tql#46]
Typescript output:

```
[11:13:13 PM] Starting compilation in watch mode...

graphql-api.ts:5839:22 - error TS2552: Cannot find name 'Query'. Did you mean 'query'?

5839   select: (t: typeof Query) => T
                          ~~~~~

graphql-api.ts:5841:56 - error TS2552: Cannot find name 'Query'. Did you mean 'query'?

5841   new Operation(name, "query", new SelectionSet(select(Query)));
                                                            ~~~~~

graphql-api.ts:5845:22 - error TS2552: Cannot find name 'Mutation'. Did you mean 'mutation'?

5845   select: (t: typeof Mutation) => T
                          ~~~~~~~~

graphql-api.ts:5847:59 - error TS2552: Cannot find name 'Mutation'. Did you mean 'mutation'?

5847   new Operation(name, "mutation", new SelectionSet(select(Mutation)));
                                                               ~~~~~~~~

graphql-api.ts:5851:22 - error TS2552: Cannot find name 'Subscription'. Did you mean 'subscription'?

5851   select: (t: typeof Subscription) => T
                          ~~~~~~~~~~~~

graphql-api.ts:5853:63 - error TS2552: Cannot find name 'Subscription'. Did you mean 'subscription'?

5853   new Operation(name, "subscription", new SelectionSet(select(Subscription)));
                                                                   ~~~~~~~~~~~~

[11:13:20 PM] Found 6 errors. Watching for file changes.
```

Full `graphql-api.ts`: https://gist.github.com/ryb73/def4d02805995f953b37f3f8405df533

Generated using command:
```
yarn --silent tql http://localhost:8080/v1/graphql > graphql-api.ts
```

I'm using a GraphQL server generated by Hasura.

It looks like it didn't generate Query, Mutation, or Subscription? Is there some configuration I need to do before running `tql`?

**Comment by ryb73** [comment]
Thanks for the library, it seems like it would make using GraphQL with TS a lot easier! Here's the dump: https://gist.github.com/ryb73/fd80d5789ffe1c21225444de35146567

**Comment by timkendall** [comment]
Thats the goal! Ah I see. Hasura is using non-standard root type names. The code generator currently assumes the standard Query, Mutation, Subscription root type names.  Should be a fairly easy fix.

**Comment by timkendall** [comment]
@ryb73 this should be fixed in the soon-to-be published `0.5.0` tag. I ran you schema through and did some light testing, seems to work.

Note: Keeping open until we resolve #47

**Comment by ryb73** [comment]
looks like it worked, thanks!
